commons/commons-lang/src/test/java/com/clicktravel/common/random/RandomsTest.java

```diff
@@ -179,6 +179,41 @@ public void shouldReturnRandomIntInRange_givenMax() {
     }
 
     @Test
+    public void shouldReturnRandomIntInRange_givenFromLessThanTo() {
+        final Set<Integer> randomInts = new HashSet<>();
+        final int lowerBound = Randoms.randomInt(200) - 100;
+        final int upperBound = lowerBound + 1 + Randoms.randomInt(200);
```

Comment:
Thinking about this more carefully, this will fail the 'all samples should not be same' if ```upperbound === lowerBound +1```. To avoid this, ensure the range is at least a reasonable size like this:
```java
final int upperbound = lowerBound + 50 + Randoms.randomInt(200);
```